src/xGadgetron/cGadgetron/include/sirf/Gadgetron/gadgetron_data_containers.h

```diff
@@ -429,7 +475,12 @@ namespace sirf {
 	*/
 	class AcquisitionsVector : public MRAcquisitionData {
 	public:
-		AcquisitionsVector(AcquisitionsInfo info = AcquisitionsInfo())
+        AcquisitionsVector(const char* filename_with_ext)
```

Comment:
I'd prefer to have this
```suggestion
        AcquisitionsVector(const std::string& filename_with_ext)
```
It will convert automatically from a `char*` (while it doesn't the other way around)